loopchain/blockchain/blocks/v1_0/block_verifier.py

```diff
@@ -1,25 +1,106 @@
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Sequence
 
 from lft.consensus.messages.data import DataVerifier
-
-from loopchain.blockchain.transactions import TransactionVersioner
+from loopchain import configure_default as conf
+from loopchain import utils
+from loopchain.blockchain.exception import BlockVersionNotMatch, TransactionOutOfTimeBound
+from loopchain.blockchain.transactions import TransactionVerifier, TransactionVersioner
+from loopchain.blockchain.types import ExternalAddress
```

Comment:
You are using `ExternalAddress` as Type hint.
Options are:
- Locate under `if TYPE_CHECKING`
- Remove quotes at type hint line.